### PR TITLE
This fixes up4 pipeline to run with single interface as well.

### PR DESCRIPTION
**Pull request by ajamshed**
Signed-off-by: Muhammad Asim Jamshed <muhammad.jamshed@intel.com>

### CODE REVIEW

File: conf/up4.bess [hunk 1]
```diff
@@ -244,19 +244,14 @@ if ports[parser.sgi_ifname].nat is not None:
...
-               check_spgwu_ip + check_gtpu_port, "gate": GTPUGate}
-sgiFastBPF.add(filters=[downlink_filter])
+else:
+    # Add Sgi filter rules, i.e.:
+    # setting filter to detect gtpu traffic
+    # and dst host x.x.x.x                         # check S/PGWU IP
+    # and udp dst port 2152                         # check GTPU port
+    check_ip = "ip"
+    downlink_filter = {"priority": -UEGate, "filter": check_ip, "gate": UEGate}
+    sgiFastBPF.add(filters=[downlink_filter])
```
Comment (krsna1729) [inline conf/up4.bess:254]: wouldnt this break N9 interface? How would N9 work in a single interface case?

Comment (krsna1729) [reply conf/up4.bess:254]: Maybe we should rename the blocks s1u/sgi as n3/n6 and add n9 block? in that we can give an cidr option to do filtering to know its n9 helping the single interface case?

Comment (ajamshed) [reply conf/up4.bess:254]: @krsna1729 , let's do this patch in next PR. Merging the current one as it is now.

File: conf/up4.bess [hunk 2]
```diff
@@ -244,19 +244,14 @@ if ports[parser.sgi_ifname].nat is not None:
...
-check_gtpu_port = " and udp dst port 2152"
-GTPUGate = 0 #ports[parser.sgi_ifname].bpf_gate()
-downlink_filter = {"priority": -GTPUGate, "filter": check_ip +
-               check_spgwu_ip + check_gtpu_port, "gate": GTPUGate}
-sgiFastBPF.add(filters=[downlink_filter])
+else:
+    # Add Sgi filter rules, i.e.:
+    # setting filter to detect gtpu traffic
+    # and dst host x.x.x.x                         # check S/PGWU IP
+    # and udp dst port 2152                         # check GTPU port
```
Comment (krsna1729) [inline conf/up4.bess:251]: comments dont reflect the filter

File: conf/up4.bess [hunk 3]
```diff
@@ -244,19 +244,14 @@ if ports[parser.sgi_ifname].nat is not None:
...
+    check_ip = "ip"
+    downlink_filter = {"priority": -UEGate, "filter": check_ip, "gate": UEGate}
```
Comment (krsna1729) [inline conf/up4.bess:253]: need to understand how "host" filter in ports.py is working when check ip filter here is broader with higher priority

**Pull request closed without merging**